### ISSUE TITLE
How to replace "spree@example.com" with customized "From" email

### ISSUE THREAD

**Issue by guangyu-shi** [spree/spree#6562]
Dear all,

Try to add the following to /config/environments/production.rb; but it doesn't work; looks spree overrides the settings of action mailer; before this, I change the "MAIL FROM ADDRESS" to 'bliss@mybliss.com' at General Settings of admin page; but it has no effects; the "from" address is still "spree@example.com" for "Reset password"  emails and order confirmation emails.

config.action_mailer.default_options = {from: 'bliss@mybliss.com'}

Any ideas?

Bliss,
--G

**Comment by guangyu-shi** [comment]
I'm running 2.4.8-stable and 2.4.2; both of them have the problem.

**Comment by sepastian** [comment]
Hi, I am using the [`spree_mail_settings` gem](https://github.com/spree-contrib/spree_mail_settings) for configuring sending mails. This gem has a `2-4-stable` branch, so it should work for you.

After adding the gem and restarting your application, you should find `Configuration > Mail Settings` on the left-hand side (not sure about the wording, I am looking at an installation with German locale right now). There you can change the *from-address*.

**Comment by mumoc** [comment]
If you're using Spree 3.0, you need to set the value on the current Store, something like
```
Spree::Store.current.mail_from_address = 'bliss@mybliss.com'
```

or with sender name
```
Spree::Store.current.mail_from_address = 'Bliss <bliss@mybliss.com>'
```

**Comment by nishant-cyro** [comment]
@guangyu-shi - Please do the honour to close this issue if above comments are helpful.

cc: @priyank-gupta @Mafi88 @damianlegawiec

**Comment by bmatmentel** [comment]
Hi,

I am reopening this, as this solution did not work for me. 

I am still finding references to the email "spree@example.com" in the mail logs, and the mail server refuses that "from email", resulting in undelivered mail. 

I added the suggested line to the initializer, I have a "default from" in the StoreMailer, and the database table for the store has been changed.

Where can I look, what am I missing?

Thanks,
bm
